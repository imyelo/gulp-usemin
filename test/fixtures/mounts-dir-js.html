<!-- build:js /mount/app.js -->
<script src="/mount/js/lib.js"></script>
<script src="/mount/js/main.js"></script>
<!-- endbuild -->