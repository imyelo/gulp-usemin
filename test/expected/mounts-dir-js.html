<script src="/mount/app.js"></script>